Packs/Anomali_ThreatStream/Playbooks/playbook-Detonate_URL_-_ThreatStream.yml

```diff
@@ -72,17 +79,22 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerrortype: ""
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
   "2":
     id: "2"
-    taskid: 8449fa75-aa65-469c-849d-89c02cb850f4
+    taskid: 40f7fd76-4554-43f3-8517-7ddb79b601d8
     type: playbook
     task:
-      id: 8449fa75-aa65-469c-849d-89c02cb850f4
+      id: 40f7fd76-4554-43f3-8517-7ddb79b601d8
       version: -1
       name: GenericPolling
```

Comment:
did we verify that this integration doesn't have a command that can replace this sub-playbook?

Reply:
Yes, the 'threatstream-get-analysis-status' command cannot be continuously run until the operation is complete.